halo2_frontend/src/plonk/permutation.rs

```diff
@@ -11,14 +10,6 @@ pub struct Argument {
     pub(crate) columns: Vec<Column<Any>>,
 }
 
-impl From<ArgumentMid> for Argument {
-    fn from(arg: ArgumentMid) -> Self {
-        Self {
-            columns: arg.columns.into_iter().map(|c| c.into()).collect(),
-        }
-    }
-}
-
```

Comment:
Because it was unused :P
Usually the flow for type conversions is Frontend -> Middleware -> Backend.  This method was doing Middleware -> Frontend; but this doesn't follow the regular flow and was unneeded.